chrome-extension/content.js

```diff
@@ -162,26 +181,31 @@ async function getCategoryDropdown(userId) {
       categoryDropdown.options.add(newOption);
     });
   } catch (error) {
-    // TODO: Handle this error?
+    // TODO: Handle error
   }
   return categoryDropdown;
 }
 
 /**
  * Add new item to databse
  */
-async function addItem() {
-  newItem = {
+async function addItem(mongoId) {
+  const newItem = {
     sourceLink: window.location.toString(),
     placesId: 'something', // TODO: get actual placesId
-    userId: currentUserId, // TODO: get actual userID
     highlight: document.getElementById('highlight').value,
     comment: document.getElementById('comment').value,
   };
 
-  const selectedCategories = getSelectedOptions('categoryDropdown');
+  if (mongoId) {
+    newItem.userId = mongoId;
+  } else {
+    alert('Please sign into Chrome to save your clipping');
```

Comment:
I noticed that this alert will appear as though it is coming from the page the user is on. This isn't ideal so maybe we should leave a todo to address it later.

Reply:
okay yeah i noticed that too. I'll leave a to-do and address it in styling pr